core/types/gen_header_json.go

```diff
@@ -69,6 +73,7 @@ func (h *Header) UnmarshalJSON(input []byte) error {
 		Extra       *hexutil.Bytes  `json:"extraData"        gencodec:"required"`
 		MixDigest   *common.Hash    `json:"mixHash"          gencodec:"required"`
 		Nonce       *BlockNonce     `json:"nonce"            gencodec:"required"`
+		BaseFee     *hexutil.Big    `json:"baseFeePerGas" rlp:"optional"`
```

Comment:
same above

Reply:
The tag `rlp:"optional"` is needed since the default value is false.